src/components/main/settings/pages/profile/mod.rs

```diff
@@ -19,6 +19,15 @@ pub fn Profile(cx: Scope<Props>) -> Element {
     let edit = use_state(&cx, || false);
     let status = use_state(&cx, String::new);
     let mp = cx.props.account.clone();
+
+    // todo: why does this not make the status persist when the settings page is reloaded/
```

Comment:
The reasoning why it doesnt persist is because the code below is not how one should be setting the status since the setting function is used mostly for internal use. The status should be set using `MultiPass::update_identity` with `IdentityUpdate::set_status_message`